rules_gapic/php/php_gapic_repositories.bzl

```diff
@@ -72,7 +72,7 @@ def php_gapic_repositories():
     _maybe(
         http_file,
         name = "php_cs_fixer",
-        urls = ["https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.9.1/php-cs-fixer.phar"],
+        urls = ["https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.16.2/php-cs-fixer.phar"],
```

Comment:
I can also downgrade this... I was experimenting with the latest version because the previous version wasn't running on PHP 7.4. IIRC, the latest version ALSO doesn't run on PHP 7.4, so this change is not necessary.